staging/src/k8s.io/kube-aggregator/pkg/registry/apiservice/etcd/etcd.go

```diff
@@ -53,6 +57,60 @@ func NewREST(scheme *runtime.Scheme, optsGetter generic.RESTOptionsGetter) *REST
 	return &REST{store}
 }
 
+var swaggerMetadataDescriptions = metav1.ObjectMeta{}.SwaggerDoc()
+
+func (c *REST) ConvertToTable(ctx context.Context, obj runtime.Object, tableOptions runtime.Object) (*metav1beta1.Table, error) {
+	table := &metav1beta1.Table{
+		ColumnDefinitions: []metav1beta1.TableColumnDefinition{
+			{Name: "Name", Type: "string", Format: "name", Description: swaggerMetadataDescriptions["name"]},
+			{Name: "Service", Type: "string", Description: "The reference to the service that hosts this API endpoint."},
+			{Name: "Available", Type: "string", Description: "Whether this service is available."},
```

Comment:
while pretty, our mechanism somehow failed if we have to turn a bool into a string. Wdyt?

Reply:
We actually do this for other columns because we're not returning a bool, we're returning a human focused message.  I.e. True and False are ok, but we *really* want to give a user the reason as well, because that answers the why.  This is parallel to how pod status, node status, etc work.

Reply:
Also the ready column is presented as `0/2` or `2/3` which allows multiple columns to be compressed, improving information density and reducing the number of follow up actions a user needs to take (yes, I know it's not ready, but HOW MUCH is it not ready, etc)